Tweet-311-Denver/tweet_311_denver_ui
# Add a map component

## What does this PR do?
- Adds a button to location that takes you to the `maps` component
- Adds `maps` component 
- Installs the react native maps package 
- Adds an image for the pin on the map
- Add functionality for grabbing the lat and long
- Adds navigation back to the `Form` 

## How to test?
- git pull changes down 
- run `npm start`
- When you run the emulator you will need to click on the word "Add Location," I wasn't sure how we wanted that to look yet
- Click on a location on the map to see the pin added 
- You will need to  `console.log()` the state to see the lat and long since it is not displayed anywhere

## Issues:
- This PR should close #12 
- The board has not been updated yet

## Screenshots:
![Screen Shot 2020-02-20 at 5 07 04 PM](https://user-images.githubusercontent.com/48968224/74991540-6d15c600-5403-11ea-872b-02c655852605.png)

## Notes:
- Not 💯 on how the flow should go yet so there is a space to add the function for updating the state in `Form`
- Also if you have any layout ideas yet me know, styling the actual map is hard but we could do anything to that bottom container 
- Let me know if you have any changes 

Thanks!
@cammac60
